src/features/cluster/cluster-modal-items.test.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
```

Comment:
The behavioural unit tests do not document behaviour as test names.

Reply:
Made some refactoring and added few more test cases as well.